Update kafka documentation with events changes (#166)
restatedev/documentation: docs/services/sdk/kafka.mdx

      return Empty.create({});
    }

    // ... Other service methods
}
```

You can mix methods receiving events with regular RPC methods. For example, assume you're implementing a `Profile` service to handle and store user profiles in Restate, where the key is the profile's unique identifier.
In the same service, you can implement a  `Get` method to retrieve the profile, and you can attach an `UpdateProfile` event handler to consume events from Kafka to update the profile.

For each event key (= Kafka record key), the events are delivered in the order in which they arrived on the topic partition.

There are some constraints for the annotated fields:

> * For event handlers input types, you must define the key type as either `string` or `bytes`.

Reviewer: ```suggestion
* The event handler's key input type must either be `string` or `bytes`.
```
